spring-resteasy/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2021, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<jboss-deployment-structure>
```

Comment:
this file must be added according to: https://github.com/resteasy/resteasy-spring/pull/71